integrations/includes/Dokan/Dashboard/Bogo.php

```diff
@@ -79,13 +90,55 @@ public function vendor_dashboard_enqueue_scripts() {
 
         $assets = include $script_assets;
 
+        wp_enqueue_style(
+            'spsg-bogo-dokan-vendor-dashboard',
+	        Helper::get_integrations_path( 'assets/build/bogo-dokan-dashboard.css' ),
+            [],
+            $assets['version'],
+        );
+
         wp_enqueue_script(
             'spsg-bogo-dokan-vendor-dashboard',
 	        Helper::get_integrations_path( 'assets/build/bogo-dokan-dashboard.js' ),
             array_merge( $assets['dependencies'], [ 'dokan-react-components' ] ),
             $assets['version'],
             true
         );
+        wp_localize_script(
+            'spsg-bogo-dokan-vendor-dashboard',
+            'spsgAdmin',
+            array(
+                'ajax_url' => admin_url( 'admin-ajax.php' ),
+                'nonce'    => wp_create_nonce( 'spsg_ajax_nonce' ),
+                'isPro'    => sp_store_growth()->has_pro(),
+            )
+        );
+        $action    = 'ajd_protected';
+        $ajd_nonce = wp_create_nonce( $action );
+
+        $script = new \StorePulse\StoreGrowth\Modules\BoGo\EnqueueScript();
+
+        wp_localize_script(
```

Comment:
Pls use [wp_add_inline_script()](https://developer.wordpress.org/reference/functions/wp_add_inline_script/) to localize the script.